corehq/apps/hqwebapp/static/cloudcare/less/formplayer-webapp/query.less

```diff
@@ -1,6 +1,7 @@
 .flex-container {
```

Comment:
Should this be a selector more specific to web apps? The `display` and `align-items` styles definitely fit with the idea of flex-container as a utility class, but background color seems possibly ambiguous.

I'm influenced by seeing `flex-container` used by dango admin templates. Spot checking them I doubt this change would make those forms look different, I'm just wary of making changes that will have side effects outside of HQ code.

Reply:
No strong feelings. I don't love classes that are just a wrapper around a single style, but I don't have a good reason for that, might be an outlier.

For the admin templates, I just realized I was grepping in a personal project earlier, not HQ. They're in my django installation, so I'd think HQ would have something similar somewhere, but we might be on a different version of django or something (they're in `python3.9/site-packages/django/contrib/admin/templates/...`)